src/main/java/pages/LoginPageFactory.java

```diff
@@ -0,0 +1,54 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class LoginPageFactory extends BasePage {
+    public LoginPageFactory(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    BasePage waitForPageOpened() {
+        return null;
+    }
+
+    @FindBy(xpath="//*[@data-test='username']")
+    WebElement usernameInput;
+    @FindBy(xpath="//*[@data-test='password']")
+    WebElement userpasswordInput;
+
+    @FindBy(id = "login-button")
+    WebElement loginBtn;
+
+    /**
+     * Login to the site
+     *
+     * @param username
+     * @param password
+     */
+    public void login(String username, String password) {
+        usernameInput.sendKeys(username);
+        userpasswordInput.sendKeys(password);
+        loginBtn.click();
+    }
+
+    /**
+     * Open login page
+     */
+    public void openPage() {
+        String loginUrl = BASE_URL  + LOGIN_URL;
+        driver.get(loginUrl);
+    }
+
+    @Override
+    protected void load() {
+
```

Comment:
try not to keep empty methods, you can add comment like:
//TODO: Implement this method

Reply:
I apologize. there was no need to add this file, all the logic from this file was moved to the LoginPage file